app/assets/stylesheets/static_pages.scss

```diff
@@ -1,19 +1,63 @@
 body {
     font-family: 'Lato', sans-serif;
 
-  #staticPages {
-    //  Styles in here
+  #staticPages{
+    text-align: center;
     .cover-img{
       width: 100%;
-      height: 500px;
+      height: 450px;
       background-image: image-url("background.png");
       background-repeat: no-repeat;
       background-size: cover;
       background-position:center;
       overflow: auto;
-      color: white
+      color: white;
+      text-align:center;
+    }
+    .title{
+      width: 100%;
+      height: 80px;
+      background-color: #41B6A6;
+      margin-top:80px;
+    }
+
+    h1,h3{
+      text-align: center;
```

Comment:
text-align you can use in bootstrap as a class: http://getbootstrap.com/css/#type-alignment (if it's inline) and http://getbootstrap.com/css/#helper-classes-center (if it's a block element)

it's more elegant I think :)